src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -60,6 +60,9 @@ internal sealed partial class Http2Connection : HttpConnectionBase, IDisposable
         //     (meaning we must assume all streams have been processed by the server)
         private int _lastStreamId = -1;
 
+        // 0 = Not opened yet, 1 = Marked as opened, 2 = Closed
+        private int _markedByTelemetryStatus;
```

Comment:
Can you use constants for 0,1,2 in the rest of the code? Will make it a little cleaner.

Reply:
Fixed